src/Sulu/Bundle/TestBundle/Kernel/SuluTestKernel.php

```diff
@@ -36,6 +36,8 @@ public function registerBundles()
             new \Sulu\Bundle\TagBundle\SuluTagBundle(),
             new \Sulu\Bundle\MediaBundle\SuluMediaBundle(),
             new \Sulu\Bundle\CategoryBundle\SuluCategoryBundle(),
+            new \Sulu\Bundle\HttpCacheBundle\SuluHttpCacheBundle(),
+            new \Sulu\Bundle\SnippetBundle\SuluSnippetBundle(),
```

Comment:
It's not really a part of this PR, but I guess when we introduce the subtree split we have to be more concise here, otherwise all the bundles will depend on each other. Or we have to find a different solution at all :-)